package/MDAnalysis/analysis/waterdynamics.py

```diff
@@ -388,79 +316,147 @@
+
+
+    Examples
+    --------
+
+    .. code-block:: python
+
+        import matplotlib.pyplot as plt
+        import MDAnalysis as mda
+        from MDAnalysis.analysis.waterdynamics import WaterOrientationalRelaxation
+
+        u = mda.Universe(waterPSF, waterDCD)
+        select = "byres name OH2"
+        wor = WaterOrientationalRelaxation(u, select)
+        wor.run(0, 7, 1)
+
+        # Plot the results
+        plt.figure(1, figsize=(18, 6))
+
+        # WOR OH
+        plt.subplot(131)
+        plt.xlabel('time / ps')
+        plt.ylabel('WOR')
+        plt.title('OH orientational relaxation')
+        plt.plot(wor.times, wor.OH, "o-")
+
+        # WOR HH
+        plt.subplot(132)
+        plt.xlabel('time / ps')
+        plt.ylabel('WOR')
+        plt.title('HH orientational relaxation')
+        plt.plot(wor.times, wor.HH, "o-")
+
+        # WOR dip
+        plt.subplot(133)
+        plt.xlabel('time / ps')
+        plt.ylabel('WOR')
+        plt.title('water dipole orientational relaxation')
+        plt.plot(wor.times, wor.dip, "o-")
+
+        plt.show()
 
 
     .. versionadded:: 0.11.0
 
     .. versionchanged:: 1.0.0
        Changed `selection` keyword to `select`
+
+    .. versionchanged:: 2.0.0
+       `t0`, `tf` and `dtmax` are renamed to start, stop, step as attributes
+       of the `run` method providing a consistent behaviour with other
+       modules.
     """
 
-    def __init__(self, universe, select, t0, tf, dtmax, nproc=1):
+    def __init__(self, universe, selection):
+        super(WaterOrientationalRelaxation, self).__init__(universe.trajectory)
         self.universe = universe
-        self.selection = select
-        self.t0 = t0
-        self.tf = tf
-        self.dtmax = dtmax
-        self.nproc = nproc
-        self.timeseries = None
+        self.selection = selection
 
-    def _repeatedIndex(self, selection, dt, totalFrames):
+    def _compare_molecules(self, selection, cur_frame, other_frame):
         """
-        Indicates the comparation between all the t+dt.
-        The results is a list of list with all the repeated index per frame
-        (or time).
-        Ex: dt=1, so compare frames (1,2),(2,3),(3,4)...
-        Ex: dt=2, so compare frames (1,3),(3,5),(5,7)...
-        Ex: dt=3, so compare frames (1,4),(4,7),(7,10)...
+        Compare the molecules in the `cur_frame` selection to the
+        `other_frame` selection and
+        select only those particles that are repeated in both frame. This is to
+        consider only those molecules that remain in the selection after
+        a certain time has elapsed. The result is a list with the indices of
+        the atoms.
+        """
+        a = set(selection[cur_frame])
+        b = set(selection[other_frame])
+        return sorted(list(a.intersection(b)))
+
+    def _get_repeated_indices(self, selection, frame):
+        """
+        Indicates the comparation between the current frame and all
```

Comment:
`comparation` -> `comparison` I think; interestingly, `comparation` does appear to be a word related to `comparison`, but largely considered obsolete from what I find online

Reply:
Huh interesting. I changed it.